app/src/components/Common/Icons/ExternalIcon.tsx

```diff
@@ -5,18 +5,24 @@ interface IconProps {
 }
 export const ExternalIcon = ({ color, className }: IconProps) => {
   return (
-    <svg
-      width="15"
-      height="18"
-      viewBox="0 0 15 18"
-      fill="none"
-      xmlns="http://www.w3.org/2000/svg"
-      className={className}
-    >
-      <path d="M14 18V11H1V18" stroke={color} strokeWidth="0.7" />
-      <path d="M12.5 14.5L10.25 15.799L10.25 13.201L12.5 14.5Z" fill={color} />
-      <circle cx="7.5" cy="4.5" r="4.15" stroke={color} strokeWidth="0.7" />
-      <path d="M2.5 14.2H10.5V14.7H2.5V14.2Z" fill={color} />
-    </svg>
+    <>
+      <svg
+        width="15"
+        height="18"
+        viewBox="0 0 15 18"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        className={className}
+      > 
+        <title>Stilisert mann med pil mot høyre</title>
```

Comment:
Er det sånn at vi kanskje burde beskrive semantikken i det vi mener med den, i staden for å beksrive korleis den ser ut? Bare en tanke, kan lite om dette faget 🤷‍♂️

Reply:
Jeg var veldig usikker på dette jeg også. Skal prøve å finne ut av det.

Reply:
"OBS! Hvis bildet formidler samme informasjon som (HTML-)tekst i nærheten, skal alt-teksten være tom, for å unngå at samme informasjon gis to ganger. Her viser bildet en dame på telefon foran en laptop, som skal vise at man kan ringe eller chatte. Samme informasjon står som ren tekst rett ved siden av bildet ("Ring eller chat med oss om økonomi"). Å gjenta dette kan oppfattes som støy, og gir ikke noen nyttig ekstra-informasjon. Bildet skal derfor ha et tomt alt-attributt." https://navikt.github.io/uu/hvordan-faa-det-til/UU-testing/manuell-testing/bilder/

Reply:
Steder der dette ikonet brukes står det ved siden av hva det indikerer. Vi må eventuelt gjøre endringer hvis ikonet skal brukes alene